### ISSUE TITLE
feature to add a title in the top navigation bar on the press page

### ISSUE THREAD

**Issue by pbastijn** [BirkbeckCTP/janeway#2712]
**Is your feature request related to a problem? Please describe.**
When people land on our press page openjournals.ugent.be, they have to read the url or go to the about page to know that they are on the UGent openjournals page. 

**Describe the solution you'd like**
Possibility to add some text to display in the top navigation bar as a title

**Describe alternatives you've considered**

**Additional context**

**Comment by ajrbyers** [comment]
@pbastijn I think the text displayed in the tab title section is the press name which you can alter here: https://openjournals.ugent.be/manager/press/ is that what was needed?

**Comment by mauromsl** [comment]
> @pbastijn I think the text displayed in the tab title section is the press name which you can alter here: https://openjournals.ugent.be/manager/press/ is that what was needed?

@ajrbyers this is to enable a custom title on the press navbar, similar to what we added for journals recently: e.g. https://openjournals.ugent.be/af/

**Comment by pbastijn** [comment]
@ajrbyers @mauromsl  yes but if possible we would prefer the title be displayed in the blue bar
